### PR TITLE
Added backers and sponsors from OpenCollective

### PR DESCRIPTION
**Pull request by piamancini**
Now sponsors & backers of your collective will show up here together with a button to your collective's public page.

### CODE REVIEW

**Comment by astorije** [comment]
@keithamus, does this come from an organization-wide decision? I won't close this myself but it does seem a bit odd to me. For example:

> Become a sponsor and get your logo on our website and on our README on Github with a link to your site.

**Comment by keithamus** [comment]
@astorije I've been setting up a Chai page on Open Collective (https://opencollective.com/chaijs) as an attempt to gather funds to cover things like domain registration. But my intention is definitely not to turn chaijs.com or the readme into a billboard. I think this is just an automated PR because I signed up to Open Collective.

Let's discuss this more as a group. I'd like to hear everyone's thoughts; pinging @meeber @lucasfcosta and @logicalparadox

**Comment by lucasfcosta** [comment]
@keithamus I understand your concerns, but I would prefer maintaining Chai through "anonymous" donations. I think that people should donate because they feel like Chai helped them in some way and not for marketing purposes. If we have costs to cover such as domain registration I would be more than happy to donate some money in order to keep things going. This project has already helped me so much that I feel like I'm just giving something back.

Instead of offering space to companies I think that if we need to raise money to maintain Chai we could use [Gratipay](https://gratipay.com/) and let people donate without receiving anything other than a sincere "Thank You" from every member of the Chai community.

Well, that's just my opinion, I'm open to hearing others' :smile:

**Comment by keithamus** [comment]
@lucasfcosta completely agree. I signed up for Open Collective to try it out and see what it has to offer. I liked the idea of regularly monthly backers but I wasn't aware (until this PR) that they expect us to advertise sponsors. 

BTW we have no urgent need for money, and the expiry of chaijs.com is in December, and a small enough fee to cover without much thought. I was mostly using it as an example of the small nominal fees a project like ours face. 

What I feel would be more interesting, with this kind of system, is to see the level of backing a project like [Mocha can get](https://opencollective.com/mochajs), and to use those funds to do more interesting things, setting up conferences and hack days, or even just reimbursing our maintainers for their work 😄

**Comment by meeber** [comment]
I'd prefer to raise funds via obnoxious merchandise: Chai t-shirts, bumper stickers, and keithamoose bobblehead dolls.

**Comment by keithamus** [comment]
So, to expand further. Systems like Gratipay, or BountySource work well for lots of projects, and I wouldn't be averse to looking at them. However they all work kind of differently...

Gratipay (until recently, as far as I know) has mostly been for individuals. It works okay, some individuals make a reasonable amount of money. The teams they have seem to make considerably less.

BountySource is a great tool to give a bit of financial incentive for people on issues that are less "exciting" to work on, like fixing old IE bugs. This rewards developers (which is good!) but doesn't really empower/reward the core team.

Open Collective seemed slightly better on the surface, it seems to have much more traction than Gratipay teams do. From the projects I've seen, like [mochajs's OpenCollective](https://opencollective.com/mochajs) [apex's OpenCollective](https://opencollective.com/apex) and [hoodie's OpenCollective](https://opencollective.com/hoodie) they seem to be getting better numbers than I can find on Gratipay. And as I mentioned above, while we have very low outgoings (engineers donate time & code, nominal fees for domain reg), having a pool of money can be a useful tool - something which it BountySource or Gratipay don't seem to provide.

**Comment by piamancini** [comment]
Hi all! Just to clarify this PR. Sorry @keithamus I thought you had seen mocha's and as I mentioned in my email these are the default backers/sponsors. You absolutely **do not** have to put sponsors and backers on your readme, **nor you have to have sponsors**. Some collectives only have backers, or even members. It's perfectly fine to just have individual backers that contribute money towards Chai just because they love the technology! Again, you can choose to include them in your readme or not. Your collective is your own, we just enable you to pool money together and spend it transparently without having to use an individual's bank account or having to create a legal entity. 

Happy to answer any questions!

**Comment by lucasfcosta** [comment]
@piamancini Thanks for clarifying! That looks good now, I just didn't like the idea of having anyone paying us for merchandising or anything like that, I think people should donate because they really WANT to contribute and they want to thank Chai for doing something for them.

@keithamus Maybe we could use this money to create shirts, stickers and organizing conferences, that sounds like a fair use of any money raised for OpenSource, I can't think of any better way to use backers' money. Let's push OpenSource forward :smile:

**Comment by meeber** [comment]
@lucasfcosta @keithamus So what do we wanna do regarding this PR? I'm thinking:

1. Backers and Sponsors badges can stay; they're neat.
1. Backers and Sponsors sections consolidated into a single "Support Us" section with simple description and link to our opencollective page, minus the list of current sponsors/backers. The existing text for backers is probably fine: "Support us with a monthly donation and help us continue our activities."

Also, since you said you're happy to answer questions, two questions for @piamancini:

1. Can I have your glasses?
1. Can I interest you in an exclusive Professor Lucas ipad decal for the low, low cost of 10 easy payments of $19.99 (plus s+h)?

**Comment by lucasfcosta** [comment]
@meeber I agree with you on both considerations. There's no problem in having these tiny little neat badges, **IMO it just would not be nice to have logos and other kind of third party marketing artifacts on our repo**, even on a separate file, a simple `Support Us` link would be enough. As I said before, **I think we should not have any bounds with donators, they should help because they feel like Chai helps them and because they want to support the whole project and not because they want something in exchange**.

TL;DR: It would be nice to accept donations, I just don't like the idea of offering marketing space (or anything other than a sincere thank you) as a reward for that (like [this](https://github.com/chaijs/chai/pull/732/files#diff-04c6e90faac2675aa89e2176d2eec7d8R72) and [this](https://github.com/chaijs/chai/pull/732/files#diff-04c6e90faac2675aa89e2176d2eec7d8R108), for example).

PS: I want one of these decals for my MacBook too! 😄

**Comment by meeber** [comment]
I don't know man, it'd be kind of unsettling to have a decal of yourself on your own MacBook. I'll talk to my distributor, maybe I can get you a vesln decal or something.

**Comment by piamancini** [comment]
dearest @meeber you can't have my glasses but I'd gladly accept the decals as a token of our nascent friendship :) Promise to send picture of my laptop proudly wearing them.

**Comment by keithamus** [comment]
> as I mentioned in my email these are the default backers/sponsors. 

Sorry @piamancini, only just got round to reading that email!

> TL;DR: It would be nice to accept donations, I just don't like the idea of offering marketing space (or anything other than a sincere thank you) as a reward for that (like this and this, for example).

I agree with this also.


So are we to change this PR to only add the badges? And we should also change the open collective page to remove the promise of README.md real estate.

**Comment by lucasfcosta** [comment]
@keithamus
> So are we to change this PR to only add the badges? And we should also change the open collective page to remove the promise of README.md real estate.

Totally agree with this. I think this is the best choice we could make :smile:

**Comment by keithamus** [comment]
@piamancini are you happy to make the changes we've agreed above?

**Comment by piamancini** [comment]
sure! Let me modify it. So just the badges of the top or also the round Become a Backer and Become a Sponsor buttons? These take you to your collective.

**Comment by piamancini** [comment]
> And we should also change the open collective page to remove the promise of README.md real estate.

What would you have it say instead?

**Comment by keithamus** [comment]
I think just the badges please @piamancini 😄

**Comment by keithamus** [comment]
> >  And we should also change the open collective page to remove the promise of README.md real estate.

> What would you have it say instead?

Could we remove the sponsor option? Just have the backer option? Is that possible please?

**Comment by keithamus** [comment]
Awesome thanks @piamancini ⭐ 

LGTM. @meeber @lucasfcosta?

**Comment by lucasfcosta** [comment]
@keithamus LGTM.
Let's do nice things with that money! :smile: 

Thanks, @piamancini

**Comment by meeber** [comment]
Since my request for @piamancini's glasses was denied I lack the necessary vision to review this PR. Proceed without me.

**Comment by keithamus** [comment]
Hey @meeber @lucasfcosta @shvaikalesh @vieiralucas - @piamancini has emailed me about a new offer for a sponsor. I'll snip the company name, for now, as I think its probably private info - if you'd like I can forward the email to you all (just lmk your email, however you can). 

> I have great news, we have a new sponsor interested in giving ChaiJS $500 month for a year. What interests <snip> is having their logo on your github readme and your website. (the readme happens automatically and here's how you can ad it to your website: https://github.com/opencollective/opencollective/wiki/Website-banner)
>
> I know there was some initial doubts about raising money in your project, so I'm reaching out before accepting the donation for you. Do you want us to accept it?
>
> best! 

I believe the consensus before was

>  I think we should not have any bounds with donators, they should help because they feel like Chai helps them and because they want to support the whole project and not because they want something in exchange.

But I wanted to float this email, rather than declining without folding all maintainers/contribs in.

**Comment by lucasfcosta** [comment]
@keithamus thanks for sharing this with us before responding! 😄 
Well, IMO we still shouldn't sell space on our README, however this brings me another tought, which, perhaps, might be more beneficial for the whole project.
Many Open Source projects and companies have sections in their READMEs and websites where they put the logos of companies which use their software and/or endorse/help the project in any way. This could be good for Chai, because we would show our users that there are big and thrusted companies using open source software we produce.

This way companies would still be able to appear on our README but not in exchange for money directly. So, if this company still wants to contribute, of course we would put their logo on our README, but under a section which contemplates other companies too, including ones that do not sponsor us.

I would love to receive donations since this would allow us to do some great and nice things such as the stickers we were planning, but I'm not very comfortable with the idea of selling space on the README, at least not on the top of it. I'm not radical about it and if you guys would like to receive donations I wouldn't see a problem, but if we can find another way to raise money that would be great too.

Also, I'd love to know which companies use our software and get in touch with them for feedback on their daily use of Chai.

Let me know what you guys think, specially @keithamus. Since he has been maintaining Chai for quite a long time and is responsible for recruiting us all, I believe he will be able to make the optimal choice for this case.

**Comment by meeber** [comment]
I don't feel strongly one way or the other

**Comment by piamancini** [comment]
Hi everyone, a couple of clarifications so you guys can make a better informed decision: 
- This sponsor is not particularly interested on the readme, but on having his logo on your site with a direct link to his website. 
- the logos on the readme don't necessarily need to be on the top. see for example: github.com/apex/apex#backers or github.com/webpack/webpack#sponsors
- What @lucasfcosta proposes is totally doable from OpenCollective's perspective. You decide where & how you put the sponsors. 
Happy to reply any additional questions.

**Comment by vieiralucas** [comment]
I was talking with @lucasfcosta and I think that his idea could be very good.
IMHO we should not be putting companies logos on chai's website without doing something like @lucasfcosta suggested.

**Comment by lucasfcosta** [comment]
Thanks for the clarification and awesome work at OpenCollective @piamancini! 😄 

So, if this company agrees, we can put his logo with a direct link on our website in a section which features companies that use our software, not necessarily companies that sponsor us with their material resources. This could go both on the bottom of our homepage and the bottom of our README, since this info is interesting to everyone using Chai and since this is good for the companies too, because they get more visibility.

So, if both the team and the company agrees, that would be great IMO.

Thanks everyone for your input, feel free to disagree and/or share your thoughts freely!

**Comment by lucasfcosta** [comment]
Hey guys, so, have we got any news on this?

**Comment by piamancini** [comment]
Hey! On our end we are wrapping up this agreement. The suggestions are cool with us. 
> This could go both on the bottom of our homepage and the bottom of our README, since this info is interesting to everyone using Chai and since this is good for the companies too, because they get more visibility.
+1

**Comment by keithamus** [comment]
Hey @piamancini quick question: does the company in question actually use and endorse chai? They are offering us money to put their logo on our website, but is this because they are actual fans of the tool, and wish to demonstrate as such?

**Comment by lucasfcosta** [comment]
Hey everyone, sorry to bother but do we have any news on this?

**Comment by piamancini** [comment]
hey everyone, the company fell through for the moment. We are working on it. 

To reply to @keithamus The company asked to support Chai and other projects in particular from the pool of https://opencollective.com/opensource. I don't think it means a special endorsement, but they do want the visibility of your site.
